module/bdev/raid/bdev_raid_sb.h

```diff
@@ -0,0 +1,98 @@
+/*   SPDX-License-Identifier: BSD-3-Clause
+ *   Copyright (C) 2022 Intel Corporation.
+ *   All rights reserved.
+ */
+
+#ifndef SPDK_BDEV_RAID_SB_H_
+#define SPDK_BDEV_RAID_SB_H_
+
+#include "spdk/stdinc.h"
+#include "spdk/util.h"
+#include "spdk/uuid.h"
+
+#define RAID_BDEV_SB_VERSION_MAJOR	1
+#define RAID_BDEV_SB_VERSION_MINOR	0
+
+#define RAID_BDEV_SB_NAME_SIZE		32
```

Comment:
Lengths of a longhorn volumes name and longhorn engine's name are 40 and 51 characters, respectively.
Can `RAID_BDEV_SB_NAME_SIZE` be `64` rather than `32`?

Reply:
Yes, we can change it. But it is about raid superblock, do you use it when you create a raid bdev with `bdev_raid_create`?

Reply:
Yeah. `bdev_raid_create` is longhorn engine's name.

Reply:
So  you call `bdev_raid_create` with the _superblock_ option right?

Reply:
No, the `name` is a must parameter for `bdev_raid_create`. It is recorded in the superblock.